openapi/resource_info.go

```diff
@@ -13,6 +13,9 @@ import (
 	"time"
 )
 
+const resourceVersionRegex = "(/v[0-9]*/)"
+const resourceNameRegex = "((/\\w*[/]?))+$"
```

Comment:
Regex pickiness... do you really want a * there?  it will match with no actual content (same for both) - should probably be a +.

Reply:
will that work for version higher than 9, e.g: 10, 11 etc? /v12/users/